autoload/codi.vim

```diff
@@ -828,3 +828,32 @@ function! codi#complete(arg_lead, cmd_line, cursor_pos)
     endif
     return sort(candidates)
 endfunction
+
+function! codi#new(ft)
+  noswapfile hide enew
+  setlocal buftype=nofile
+  setlocal bufhidden=hide
+
+  call codi#run(0, a:ft)
```

Comment:
When running without any arguments, `:CodiNew` gives the following error, which is a little cryptic:

`E471: Argument required`

Can we add some error handling, like with `:Codi`?

```vim
    if empty(a:filetype)
      return s:err('Cannot run Codi with empty filetype.')
    else
      return s:err('No Codi interpreter for '.a:filetype.'.')
    endif
```

We could make things even simpler by keeping the specification from the original `:Codi` command:

```
If {filetype} is not specified, the buffer's 'filetype' is used instead.
```